assets/js/handlebars/templates/settings-modal-body.handlebars

```diff
@@ -3,7 +3,7 @@
     <label class="switch-button-container">
       <div>
         <span>Night mode</span>
-        <p>Switch to a dark mode version of the website.</p>
+        <p>Use the dark mode version of this website.</p>
```

Comment:
```suggestion
        <p>Switch the documentation UI to dark mode.</p>
```

WDYT?

Reply:
The above is an idea to make it more concise and remove the ambiguity as you proposed. :)

Reply:
Using the word "Switch" makes no sense if I am already using dark mode, hence the use of "Use":

```suggestion
        <p>Use the documentation UI in a dark theme</p>
```

> to dark mode

From a grammatical point of view, "to dark mode" implies that our dark version is called "dark mode". Since we are giving a description of Night mode, "a dark mode" or "a dark theme" would be more apt.